API.md

```diff
@@ -3394,6 +3394,18 @@ following options:
 - `noSniff` - boolean controlling the 'X-Content-Type-Options' header. Defaults to `true` setting
   the header to its only and default option, `'nosniff'`.
 
+- `referrer` - controls the ['Referrer-Policy'](https://www.w3.org/TR/referrer-policy/) header value:
+    - `false` - the 'Referrer-Policy' header will not be sent with responses. This is the default value.
+    - `''` - empty string indicating that the Referrer-Policy will be defined elsewhere.
+    - `'no-referrer'` - never include the referrer header.
```

Comment:
@geek this is question more than feedback on PR - what is the difference here between `false` and `'no-referrer'`?

Reply:
The description is misleading...